src/utils/schematron/oscal-profile-sources.sch

```diff
@@ -6,6 +6,7 @@
   
   <sch:ns uri="http://csrc.nist.gov/ns/oscal/1.0" prefix="oscal"/>
 
+<!-- (This XSLT is broken until profile resolution is (re)stabilized -->
```

Comment:
@wendellpiez Is this still broken? If so, we might want to hold off on this PR. Also, we might want to add this to the content validation CI/CD workflow for profiles.

Reply:
The Schematron has been broken for a while, it's really only the comment saying so that is new with the PR. At the same time, I think there's a fair chance that the necessary repair will be pretty straightforward. Agreed that the check could be run under CI. In principle any profile document ought to pass when it is correctly linked to its imported catalogs (or profiles once that is connected back up).

Reply:
Hi again, actually I should qualify ... the suspect part of the Schematron is only its dependency on an XSLT for profile resolution. This does not make it completely useless but it is less useful that otherwise. Given a profile resolver (at least, a copy of the prototype it was built with, or a replacement) it might work fine. Probably the comment was added because in meeting the other requirement, I found that its link to the resolver wasn't operable (but I didn't have time to dig further).

Fine to push the PR back, inasmuch as the problem it addresses is not acute, and the data error that gave rise to it, has been corrected.